src/main/resources/config/en.yml

```diff
@@ -94,29 +99,37 @@ Experiment_MCDiscordReserializer_ToMinecraft: false
 Experiment_MCDiscordReserializer_InBroadcast: false
 
 # Other
+# Whether to cancel a console command if it can't be logged
 CancelConsoleCommandIfLoggingFailed: true
+# A language code to force DiscordSRV to use; See https://github.com/DiscordSRV/DiscordSRV/tree/master/src/main/resources/config for a list of available codes
 ForcedLanguage: none
```

Comment:
Should mention that this only effects internal messages and not ones in messages.yml